bindings/hyperdrivepy/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "hyperdrivepy"
+version = "0.14.0"
+requires-python = ">=3.7"
```

Comment:
3.7?  can we bump? do we care?

Reply:
we don't care too much; I think it was pyo3 that set this limit. Once we bump `agent0` to 3.11 we can bump this one too and remove all `from __future__` imports.